src/intents.cs

```diff
@@ -1174,6 +1154,25 @@ interface INDoubleResolutionResult {
 
 	[Introduced (PlatformName.iOS, 10, 0)]
 	[Unavailable (PlatformName.MacOSX)]
+	[DisableDefaultCtor]
+	[BaseType (typeof (INIntentResolutionResult))]
+	interface INDateComponentsResolutionResult {
+
+		[Static]
+		[Export ("successWithResolvedDateComponents:")]
+		INDateComponentsResolutionResult GetSuccess (NSDateComponents resolvedDateComponents);
```

Comment:
In general we keep the same names as Apple, unless they are incorrect (wrong/typos) or very confusing. It's globally more consistent (e.g. newer API In future SDK will likely use the names, but the _binder_ might not know we changed them in older API).